release.sh

```diff
@@ -1,9 +1,9 @@
 # This if for Release Health. It sets a release according to Calendar Versioning and gets redeployed every day
 # In sentry-demos/tracing we did not manage and autodeploy this daily, so you were stuck with the same Release for eternity until you redeployed
 # /react and /flask both have a run.sh script that utilizes this code
-day=$(date +%d)
-month=$(date +%-m)
-year=$(date +%y)
+day=$(date -u +%d)
```

Comment:
@realkosty you may have explained to me in Slack, but could you document in the PR what the before - vs - after is of the command here? What is the output of `$(date +%d)` versus `$(date -u +%d)`, so we can understanding?

Is it anything viewable in the UI? If you could put links to the releases. Otherwise just put the value of what `day` is.

Reply:
thanks, I added a comment explaining it's UTC

added some testing to PR description.
I will build and run the whole thing once I have a moment too

Reply:
updated testing with sample event